cl/users/templates/includes/search_alert_webhook_dummy.txt

```diff
@@ -0,0 +1,145 @@
+{
+   "results": [
+        {
+            "absolute_url": "/opinion/2812209/obergefell-v-hodges/",
+            "attorney": "",
+            "author_id": null,
+            "caseName": "Obergefell v. Hodges",
+            "caseNameShort": "Obergefell",
+            "citation": [
+                "192 L. Ed. 2d 609",
+                "135 S. Ct. 2584",
+                "2015 U.S. LEXIS 4250",
+                "576 U.S. 644"
+            ],
+            "citeCount": 432,
+            "cites": [
+                87116,
```

Comment:
I think I'd leave the `cites` key blank just to keep it simple for folks. Same in the cURL example.

Reply:
Updated.